lib/bridge/IrcHandler.js

```diff
@@ -443,7 +443,10 @@ IrcHandler.prototype.onMetadata = Promise.coroutine(function*(req, client, msg)
         let response = yield this.ircBridge.getAppServiceBridge().getIntent().createRoom({
             createAsClient: false,
             options: {
-                name: "IRC Admin Room",
+                name: `${client.server.domain} Admin`,
```

Comment:
How about keeping it more informal perhaps, and something like "Your Freenode Connection"?